integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -0,0 +1,98 @@
+use itertools::Itertools;
+use near_async::test_loop::data::TestLoopData;
+use near_async::time::Duration;
+use near_chain_configs::test_genesis::TestGenesisBuilder;
+use near_o11y::testonly::init_test_logger;
+use near_primitives::epoch_manager::EpochConfigStore;
+use near_primitives::shard_layout::ShardLayout;
+use near_primitives::types::{AccountId, ShardId};
+use near_primitives::version::{ProtocolFeature, PROTOCOL_VERSION};
+use std::collections::BTreeMap;
+use std::sync::Arc;
+
+use crate::test_loop::builder::TestLoopBuilder;
+use crate::test_loop::env::TestLoopEnv;
+use crate::test_loop::utils::ONE_NEAR;
+
+/// Stub for checking Resharding V3.
+/// After uncommenting panics with
+/// StorageInconsistentState("Failed to find root node ... in memtrie")
+#[test]
+// #[ignore]
```

Comment:
nit: can remove this "ignore"

Reply:
oh, I need to comment that back. I couldn't make the test pass